src/PtrArrays.jl

```diff
@@ -87,4 +87,9 @@ Base.@propagate_inbounds function Base.setindex!(p::PtrArray, v, i::Int)
     p
 end
 
+# Strided array interface
+Base.strides(p::PtrArray) = Base.size_to_strides(1, size(p)...)
```

Comment:
```suggestion
Base.strides(p::PtrArray) = accumulate(*, size(p)[begin:end-1])
```
Base.size_to_strides is internal (and also an IMO pretty ugly way of writing `accumulate(*, _[begin:end-1])`, PR to base forthcoming)

Reply:
I definitely agree, but it does require `(1, accumulate(*, size(p)[begin:end-1]...)`...

Reply:
It also seems like `accumulate` and `cumprod` are not backwards compatible with Julia 1.0, so I'll revert that change for now.

Reply:
How about this most concise definition:

```suggestion
```

See: JuliaLang/julia#54558